docs/DeveloperGuide.md

```diff
@@ -299,56 +344,54 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 **Aspect: How contacts are saved with multiple arguments:**
 
-* **Alternative 1 (current choice):** Only include people who contain all the specified contact details 
+* **Alternative 1 (current choice):** Only include people who contain all the specified contact details
     * Pros: Intuitive feature. Similar to a Filter function in popular apps today.
     * Cons: Requires you to be familiar of the people in your contact list.
 
 * **Alternative 2:** Include people who contain at least one of the specified contact details.
-    * Pros: Good for users who want to broadly search for eligible friends 
+    * Pros: Good for users who want to broadly search for eligible friends
     * Cons: Not very intuitive
 
-### Delete by keywords
+### Delete multiple persons
 
 #### Implementation
 
-The Delete By Keyword mechanism will delete contacts specified by a given set of keywords. Any contacts containing any of the specified keywords will be deleted.
+The deleting multiple person mechanism will delete contacts specified by a given set of keywords. Any contacts containing all the specified keywords will be deleted.
```

Comment:
Could perhaps bold the word "all" here to emphasise that our application deletes by "AND" and not "OR"